### PR TITLE
689: Footer added

### PR DESCRIPTION
**Pull request by Mdemenko1**
### What (if any) features are you implementing?

Work for the issue [689](https://github.com/Gary-Community-Ventures/benefits-calculator/issues/689)

Adding footer in midBlue

image

### CODE REVIEW

File: src/Components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Paper from '@mui/material/Paper';
+import './Footer.css';
+
+const Footer = () => {
+  return (
+    <nav>
```
Comment (CalebPena) [inline src/Components/Footer/Footer.js:6]: Can we use a `footer` element instead?

**Review by CalebPena — CHANGES_REQUESTED** [review]
It looks like the footer is blocking the back and continue buttons. Could you make it so it is the last element on the screen, but also at the bottom of the page. [This article](https://www.freecodecamp.org/news/how-to-keep-your-footer-where-it-belongs-59c6aa05c59c/) might be helpful.

**Comment by Mdemenko1** [comment]
@sydney-devine 

Do we want to footer to stay on the page all the time or only see it when you are scrolling to the bottom of the page?

All the time
image



After scrolling

https://github.com/Gary-Community-Ventures/benefits-calculator/assets/65931890/e7b17e2a-ed9b-4f25-938d-c0a3bd85bae4

**Comment by CalebPena** [comment]
@Mdemenko1 We want the footer to be at the bottom of the screen after scrolling, but we don't want it in the middle of the screen if the page is shorter.

**Comment by Mdemenko1** [comment]
> @Mdemenko1 We want the footer to be at the bottom of the screen after scrolling, but we don't want it in the middle of the screen if the page is shorter.

Current behavior, footer appears on the end of the page after scrolling


https://github.com/Gary-Community-Ventures/benefits-calculator/assets/65931890/b3f532f7-dd8e-4728-86cb-449ffc3e3dc3

**Comment by CalebPena** [comment]
@Mdemenko1 Sorry for the confusion, we want the footer to be at the very bottom if the page does not take up the full screen, but if the page does take up the whole screen, then we want it to be at the bottom after scrolling. So in the video we showed, we would want the footer to be at the bottom of the screen the whole time.

**Comment by Mdemenko1** [comment]
@CalebPena please check out new update for the footer when you have a minute, thank you

**Comment by CalebPena** [comment]
@Mdemenko1 [This article](https://www.freecodecamp.org/news/how-to-keep-your-footer-where-it-belongs-59c6aa05c59c/) explains what we want to do. There are 2 cases. If the content is shorter than the screen height, then we want the footer to be at the bottom of the page. If the content is longer than the screen height, then we want the footer to be after the content.

**Comment by sydney-devine** [comment]
> @sydney-devine
> 
> Do we want to footer to stay on the page all the time or only see it when you are scrolling to the bottom of the page?
> 
> All the time image
> 
> After scrolling
> 
>  Screenshare.-.2024-01-09.10_12_56.PM.mp4

@Mdemenko1 Only when you scroll to bottom of the page. Sorry for the delayed response from being out last week!

File: src/App.css [hunk 2]
```diff
@@ -90,7 +90,14 @@
 
 .main-max-width {
   max-width: var(--main-max-width);
-  margin: auto;
+  height: auto !important;
```
Comment (CalebPena) [inline src/App.css:93]: I don't think that you need this line unless there is an edge case that I am missing.

File: src/App.css [hunk 3]
```diff
@@ -90,7 +90,14 @@
...
+  margin: 0 auto -5rem;
+  min-height: 100%;
```
Comment (CalebPena) [inline src/App.css:95]: This line is overridden by the next line.

**Review by CalebPena — APPROVED** [review]
Good job on this. There are 2 lines of CSS that I think you can delete because they are not doing anything.

**Comment by Mdemenko1** [comment]
@CalebPena, I think its ready for merging

**Review by CalebPena — APPROVED** [review]

**Comment by CalebPena** [comment]
@Mdemenko1 Changing the branch to be merged into `dev`.

**Pull request merged**